docs/changelog_de.md

```diff
@@ -8,6 +8,7 @@
 * Fix [#147](https://g1cp.org/issues/147): Die Rüstung "Crawler-Plattenrüstung" heißt nun korrekt "Crawlerplatten-Rüstung".
 * Fix [#148](https://g1cp.org/issues/148): Die Rüstung "antike Erzrüstung" heißt nun korrekt "Antike Erzrüstung".
 * Fix [#149](https://g1cp.org/issues/149): Die Rüstung "verbesserte Erzrüstung" heißt nun korrekt "Verbesserte Erzrüstung".
+* Fix [#172](https://g1cp.org/issues/172): Die Schriftrolle "Kalom's Rezept" heißt nun korrekt "Kaloms Rezept".
```

Comment:
Two notes on that:
- Should this not be listed in the story section of the changelog? This is a story item, which is unlikely to appear in mods that tell a different story.
- I get why you added the word "Schriftrolle" and I see how that is an accurate description, but I wonder if this will lead to misunderstandings as "Schriftrolle" usually exclusively refers to spell scrolls in the game. I, for one, was very confused at first when I read it. We could refer to it as "Dokument" or "Schriftstück" instead. What do you think?